src/form/form-label.style.tsx

```diff
@@ -38,4 +38,5 @@ export const ErrorMessage = styled(Text.H6)`
 export const Subtitle = styled(Text.XSmall)<LabelProps>`
```

Comment:
why dont use `Text.BodySmall` then? that is 1rem. Unless u want to keep the line height and letter spacing of `XSmall`?